src/main/java/view/Input.java

```diff
@@ -0,0 +1,41 @@
+package view;
+
+import java.util.Scanner;
+
+public class Input {
+
+
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static String[] getNames(){
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String str;
+        String[] names;
+
+        while(true) {
+
+            str = scanner.nextLine();
+
+            names = str.split(",");
+
+            if(nameIsGood(names))break;
+            System.out.println("이름은 5글자 이내로 다시 작성해주세요");
+
+        }
+
+        return names;
+    }
+
+    public static int getCount(){
+        System.out.println("시도할 회수는 몇회인가요?");
+        return scanner.nextInt();
+    }
+
+    public static boolean nameIsGood(String [] names){
+        for(var name: names){
+            if(name.length()>=5)return false;
```

Comment:
```suggestion
            if (name.length() >= 5) {
                return false;
            }
```
코드 컨벤션을 신경 써주세요. 앞으로 동호님이 작성하실 코드는 동호님 혼자서만 보는 코드가 아닙니다!

협업하는 동료를 위해서라도, `반드시` 컨벤션은 맞춰서 제출해주세요 ㅎㅎ

모두 신경쓰기 어렵다면, 인텔리제이 자동정렬을 적극 활용해주시면 좋아요 !

Reply:
의식적인 습관 드리겠습니다!!